ckanext/fluent/validators.py

```diff
@@ -23,7 +23,8 @@
 
 @scheming_validator
 def fluent_core_translated_output(field, schema):
-    assert field['field_name'].endswith(LANG_SUFFIX), 'Output validator "fluent_core_translated" must only used on a field that ends with "_translated"'
+    assert field['field_name'].endswith(LANG_SUFFIX) or field[
```

Comment:
Moet field['field_name'] =='qual_ass' ? Is het niet beter om de fluent structure te volge en "_translated" toe te voegen aan field_name van 'qual_ass'

Reply:
Done!